packages/graphback-datasync/src/util.ts

```diff
@@ -39,3 +44,59 @@ export const DataSyncFieldNames = {
   lastUpdatedAt: '_lastUpdatedAt',
   deleted: '_deleted'
 }
+
+
+
+export interface DataSyncModelConflictConfig {
+  enabled: boolean
+  conflictResolution?: ConflictResolutionStrategy
+}
+
+export interface DataSyncModelConfigMap {
+  [modelName: string]: DataSyncModelConflictConfig
+}
+
+export interface ConflictResolutionStrategy {
+  resolveUpdate(conflict: ConflictMetadata): any
+  resolveDelete(conflict: ConflictMetadata): any
+}
+
+export const ServerSideWins: ConflictResolutionStrategy = {
+  resolveUpdate(conflict: ConflictMetadata): any {
+    const { serverData, serverDiff, clientDiff } = conflict;
+
+    if (serverData[DataSyncFieldNames.deleted] === true) {
+      throw new ConflictError(conflict);
+    }
+
+    const resolved = Object.assign(serverData, clientDiff, serverDiff);
+
+    return resolved
+  },
+  resolveDelete(conflict: ConflictMetadata): any {
+
+    throw new ConflictError(conflict);
+  }
+}
+
+export const ClientSideWins: ConflictResolutionStrategy = {
+  resolveUpdate(conflict: ConflictMetadata): any {
+    const { serverData, clientDiff } = conflict
+
+    const resolved = Object.assign(serverData, clientDiff);
+
+    if (serverData[DataSyncFieldNames.deleted] === true) {
+      resolved[DataSyncFieldNames.deleted] = false;
+    }
+
+    return resolved;
+  },
+  resolveDelete(conflict: ConflictMetadata): any {
+    const { serverData, clientData } = conflict;
+
+    if (serverData[DataSyncFieldNames.deleted] === true || clientData[DataSyncFieldNames.deleted] === true) {
```

Comment:
We are missing deletion integration tests.

[Question] want to delete from client (on server this changed) - client wins should force delete. With current handlers it looks like deletion support makes no sense at all (we always return error to client - in both strategies.